security/security_policy_compliance/policy-checks.sh

```diff
@@ -38,6 +44,10 @@ start "Checking Time Machine Backup Disk Encryption"
 # Replace spaces with three underscores so we can use spaces as a delimiter in an array
 TIME_MACHINE_VOLUMES=$(tmutil destinationinfo | grep -e "Name" | sed 's/Name.*: //' | sed -e "s/ /___/g")
 
+if [ "${TIME_MACHINE_VOLUMES}" == "" ]; then
+  warning "No backup disks configured with Time Machine."
```

Comment:
Can we give this URL as help similar to the `fail` outcome?

https://github.com/sparkbox/standard/blob/1ca790bf2e5d553264b5d746ab3a9c91a6ffb766/security/security_policy_compliance/timemachine.md#first-time-setup

Reply:
Ah, good catch! I just updated this in aa8b0f7eee3d77a925a203ae1ce7dec8854e514d. How's it looking now?

Reply:
Ah, good catch! I updated it in fea2d79f5031a8d45bd5a5469f68f947d8dc8d13. How's it looking now?